### DISCUSSION TITLE
memoize function

### DISCUSSION THREAD

**Discussion by DaSchTour** [marpple/FxTS#194] [Ideas]
I found this while searching for an alternative to lodash and actually most of the functions I use from lodash are also available. Something that I miss most is memoize. Would be nice see this implemented here.

**Comment by shine1594** [comment]
I agree that the memoize function would be useful. Our team will review it positively. Thank you for your suggestion.

**Comment by shine1594** [comment]
The `memoize` function was added in version 0.14.0.

**Reply by shine1594** [reply]
https://github.com/marpple/FxTS/blob/c938202e8ca832eb1cc3b82afab70886ef5f6c1e/src/memoize.ts#L35
